docs/_data/sidebars/jupedsim_sidebar.yml

```diff
@@ -0,0 +1,165 @@
+# This is your sidebar TOC. The sidebar code loops through sections here and provides the appropriate formatting.
+
+entries:
+- title: sidebar
+  product: JuPedSim
+
+  folders:
+
+  - title:
+    output: pdf
+    type: frontmatter
+    folderitems:
+    - title:
+      url: /titlepage.html
+      output: pdf
+      type: frontmatter
+    - title:
+      url: /tocpage.html
+      output: pdf
+      type: frontmatter
+
+  - title: Overview
+    output: web, pdf
+    folderitems:
+
+    - title: Home
+      url: /index.html
+      output: web
+      type: homepage
+
+    - title: Introduction
+      url: /jupedsim_introduction.html
+      output: web, pdf
+
+    - title: Support
+      url: /jupedsim_support.html
+      output: web, pdf
+
+    - title: Disclaimer
+      url: /jupedsim_disclaimer.html
+      output: web, pdf
+
+  - title: Installation
+    output: web, pdf
+    folderitems:
+
+    - title: Dependencies
+      url: /jupedsim_requirements.html
+      output: web, pdf
+
+
+    - title: Install JuPedSim on Mac
+      url: /jupedsim_install_on_mac.html
+      output: web, pdf
+
+    - title: Install JuPedSim on Linux
+      url: /jupedsim_install_on_linux.html
+      output: web, pdf
+
+    - title: Install JuPedSim on Windows
+      url: /jupedsim_install_on_windows.html
+      output: web, pdf
+
+  - title: Tests
+    output: web, pdf
+    folderitems:
+    - title: RiMEA
+      url: /jpscore_tests_rimea.html
+      output: web, pdf
+
+    - title: Juelich
+      url: /jpscore_tests_juelich.html
+      output: web, pdf
+
+  - title: Simulation
+    output: web
+    folderitems:
+
+    - title: jpscore - Introduction
```

Comment:
Maybe the linkname should be Introduction only. `jpscore - Introduction` breaks the menu structure.